### DISCUSSION TITLE
Again - can't upload a certificate

### DISCUSSION THREAD

**Discussion by OpenK54** [ZoeyVid/NPMplus#294] [Q&A] [answered]
I get this error message: Upload failed: Certificate is not valid (Command failed: openssl x509 -in /tmp/78e7346c-8c3b-4ae2-989b-9d26564254be/tmp -subject -noout Error relocating /usr/bin/openssl: OSSL_CMP_MSG_update_recipNonce: symbol not found )

But I know that my certificate is valid. I have a confirmation from DigiCert. To be sure the key is correct I reissued the certificate with a new key. Still no music. What can I do? Why can't we just upload a pfx file and be done with it?

**Comment by Zoey2936** [comment]
I will push a fix in a few minutes

**Reply by OpenK54** [reply]
Great to read this. Thanks!

**Reply by Zoey2936** [reply, marked as answer]
please try again with the latest version

**Comment by OpenK54** [comment]
It worked!:-)

**Reply by Zoey2936** [reply]
perfect!

**Comment by OpenK54** [comment]
Sorry, but there still seams to be a bug. One time it worked perfectly. But with my other instances I had the problem that it uploaded the certificate but set the current date and time as expiration time.

**Reply by Zoey2936** [reply]
I had this bug also, but it should be fixed

**Reply by Zoey2936** [reply]
![grafik](https://github.com/ZoeyVid/nginx-proxy-manager/assets/75573284/231592cd-939d-4786-87b4-56aedd70febf)

**Reply by OpenK54** [reply]
I found out what the problem is. It seems to be a browser issue. With edge everything works perfectly wheras with firefox I see upload issues. I have no idea why. Maby you can reproduce it and finde the root cause.

**Reply by Zoey2936** [reply]
do you use addons? I use firefox dev and it works

**Reply by OpenK54** [reply]
only

![grafik](https://github.com/ZoeyVid/nginx-proxy-manager/assets/132947170/84ae1868-2ddf-4028-86c9-e45c3c45e627)
